api/api.py

```diff
@@ -344,12 +344,14 @@ def load(path: str, *args):
     hi() # prints "Hello world!"
   """
 
-def local(command: str, quiet: bool = False) -> Blob:
+def local(command: str, quiet: bool = False, command_bat: str = "") -> Blob:
   """Runs a command on the *host* machine, waits for it to finish, and returns its stdout as a ``Blob``
 
   Args:
     command: Command to run.
     quiet: If set to True, skips printing output to log.
+    command_bat: The command to run, expressed as a Windows batch command executed
+      with ``cmd /S /C``. Takes precedence over the ``command`` parameter on Windows.
```

Comment:
What happens if you specify it in a non-Windows context?

Reply:
What happens if you pass `command` in Windows?

Reply:
clarified this a bit